prism/src/main/java/org/apache/falcon/service/BacklogMetricEmitterService.java

```diff
@@ -206,6 +216,7 @@ public synchronized void onSuccess(WorkflowExecutionContext context) throws Falc
                 metrics.remove(new MetricInfo(DATE_FORMAT.get().format(date), context.getClusterName()));
                 if (metrics.isEmpty()) {
                     entityBacklogs.remove(entity);
+                    publishBacklog((Process) entity, context.getClusterName(), 0L);
```

Comment:
Understand from @PraveenAdlakha that we are doing an eager drop as we are removing entityBacklog entry and the emitter task will have no handle of this process. 

So, the change is fine.